core/templates/dev/head/pages/library/Library.js

```diff
@@ -34,7 +34,7 @@ oppia.controller('Library', [
   'AlertsService', 'ConstructTranslationIdsService',
   'LearnerDashboardActivityIdsObjectFactory',
   'LearnerDashboardIdsBackendApiService', 'LearnerPlaylistService',
-  'SearchService',
+  'PageTitleService', 'SearchService',
```

Comment:
Can you remove unwanted dependencies? (This is unrelated changes to this PR, asking this as I see lots of unwanted dependencies here.)

Reply:
Okay. Should they be removed in this PR?

Reply:
Yes! (as it's just simple cleaning stuff.)

Reply:
After going through them, I removed most unused dependencies then I noticed that they failed Travis CI tests meaning that they're actually being used probably for tests.

Reply:
I think that's happening because you have removed the imported .js file from the script tag. (not because of removing unwanted dependencies.)